lib/basedriver/helpers.js

```diff
@@ -12,14 +12,20 @@ import request from 'request';
 
 const ZIP_EXTS = ['.zip', '.ipa'];
 
-async function configureApp (app, appExt) {
+async function configureApp (app, appExt, mountroot="Volumes", windowsShareUserName="", windowsSharePassword="") {
   if (_.isNull(app) || _.isUndefined(app)) {
     // immediately shortcircuit if not given an app
     return;
   }
 
   let newApp = null;
   let shouldUnzipApp = _.includes(ZIP_EXTS, path.extname(app));
+  let shouldCopyFromWindowsNetworkShare = _.startsWith(app, "\\");
+
+  if (shouldCopyFromWindowsNetworkShare) {
```

Comment:
again, a small thing. Maybe this is more readable, but since we're only using this variable in one place, we could more succinctly write this as:

```js
// check if we're copying from a windows network share
if (_.startsWith(app, "\\")) {
```